packages/util/src/pipe.ts

```diff
@@ -1,5 +1,22 @@
+/**
+ * Same as pipe but for asynchronous functions:
+ * asynvPipe(finc1,func2,...funcN)(start)  executes all functions in sequence awaiting the result and piping the response through in other words await funcN(... await func2(await func1(start))...)
+ * @deprecated We will remove this in version 2.0 of the SDK.
+ *
+ * @param fns - a list of asynchronous functions to be executed in sequence
+ * @param start - start value for the first function in the list
+ * @returns the reponse of the last function in the list.
+ */
 export const asyncPipe = (...fns) => (start: any): Promise<any> =>
```

Comment:
This can also be removed?

Reply:
we do not use it, but it was there before and customers could use it

Reply:
maybe we can also deprecate this for 2.0